CMakeLists.txt

```diff
@@ -18,6 +18,8 @@ GET_FILENAME_COMPONENT(depsAbs
     "${DEPS_PATH}" REALPATH
     BASE_DIR "${CMAKE_BINARY_DIR}")
 
+# SET(CUDA_TOOLKIT_ROOT_DIR /usr/local/cuda-9.0)
```

Comment:
We should really test things on a CUDA target before merging, we need to ensure everything is smooth. Let's align on this.

Reply:
I commented it out after an experiment where I extracted the CUDA SDK from its container and used it to build RedisAI, which went well. The more challenging problem is the runtime problem, which apparently requires some fiddling with nVidia drivers.
I kept the comment as a reminder/reference.